tools/ui-components/src/alert/alert.tsx

```diff
@@ -35,7 +40,12 @@ export function Alert({
 
   return (
     <div className={classes} role='alert'>
+      {canDismiss ? (
+        <CloseButton onClick={onDismiss} label={closeLabel ?? 'Close Alert'} />
```

Comment:
Should we make the default fallback translatable?

Reply:
is `useTranslation` can be imported? 🤔

Let me check in the morning, if not we may need to discuss this with Ahmad.

Reply:
I don't think it's possible to use `useTranslate`.

For now, I have removed the value, and made it if someone use `onDismiss`, they have to provide a value, so it isn't possible to have `onDismiss`, without `closeLabel` and vise-versa.

This doesn't solve the issue of untranslatable strings in the ui-component, but this forces us to provided a string (translatable or not) in the `/learn`, when using the component.

We will still have to discuss this with Ahmad, on how he want this untranslatable strings to be tackled.

Reply:
I will mention this in today meeting, and open issue about what we come up with.